### ISSUE TITLE
Vue app creation failing

### ISSUE THREAD

**Issue by silloyd** [single-spa/create-single-spa#164]
I'm having trouble creating a vue project using create-single-spa.

Looks like if I try to use the current dir for the project, then it uses '.' as the project name which gets rejected as an invalid name

```
# create-single-spa
(node:14199) ExperimentalWarning: The fs.promises API is experimental
? Directory for new project .
? Select type to generate single-spa application / parcel
? Which framework do you want to use? vue
? Organization name (use lowercase and dashes) en
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: projectName must use lowercase and dashes!
    at SingleSpaVueGenerator.runVueCli (/root/.nvm/versions/node/v10.15.3/lib/node_modules/create-single-spa/node_modules/generator-single-spa/src/vue/generator-single-spa-vue.js:55:13)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Emitted 'error' event at:
    at Immediate.setImmediate (/root/.nvm/versions/node/v10.15.3/lib/node_modules/create-single-spa/node_modules/yeoman-generator/lib/index.js:1007:20)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
#
```

If I go up a dir and point into the folder, it passes the projectName check but then just exits without setting up the project

```
# create-single-spa
(node:2439) ExperimentalWarning: The fs.promises API is experimental
? Directory for new project reporting
? Select type to generate single-spa application / parcel
? Which framework do you want to use? vue
? Organization name (use lowercase and dashes) en
#
```

**Comment by filoxo** [comment]
This is expected and has always been the behavior for generating Vue apps. The latest update was supposed to handle the directory parameter a little better and derives the projectName from there, but I have tested this out and verified that it is indeed failing to create the vue app. I can work on fixing this today. 

As a workaround, using a relative path as the `dir` option (eg. `npx create-single-spa ./reporting`) works correctly.

**Comment by filoxo** [comment]
I'm also open to updating this so that the projectName is explicitly asked for instead. What do you think? Also cc @joeldenning.

**Comment by silloyd** [comment]
Thanks for the quick response! Using the relative path does indeed work - so thanks for that.

I think deriving the project name from the directory is fine, but perhaps explicitly asking for it would make the flow more consistent with the other frameworks.

**Comment by joeldenning** [comment]
Running create-single-spa on the current directory should be supported. I think that prompting for the projectName separately in that case makes the most sense.

**Comment by filoxo** [comment]
I published this in [create-single-spa@1.12.3](https://github.com/single-spa/create-single-spa/releases/tag/v1.12.3).
